http/src/main/resources/init-resources/init-actions.sh

```diff
@@ -276,34 +275,19 @@ END
       gsutil cp ${CUSTOM_ENV_VARS_CONFIG_URI} /var
     fi
 
-    if [ ! -z ${SERVICE_ACCOUNT_CREDENTIALS} ] ; then
-      echo "GOOGLE_APPLICATION_CREDENTIALS=/var/${SERVICE_ACCOUNT_CREDENTIALS}" > /var/google_application_credentials.env
-    else
-      echo "" > /var/google_application_credentials.env
-    fi
-
-    # Install RStudio license file, if specified
-    if [ ! -z "$RSTUDIO_DOCKER_IMAGE" ] ; then
-      STAT_EXIT_CODE=0
-      gsutil -q stat ${RSTUDIO_LICENSE_FILE} || STAT_EXIT_CODE=$?
-      if [ $STAT_EXIT_CODE -eq 0 ] ; then
-        echo "Using RStudio license file $RSTUDIO_LICENSE_FILE"
-        gsutil cp ${RSTUDIO_LICENSE_FILE} /etc/rstudio-license-file.lic
-      else
-        echo "" > /etc/rstudio-license-file.lic
-      fi
-    fi
+    echo "" > /var/google_application_credentials.env
```

Comment:
Maybe we can remove the empty file as well if we remove it from [jupyter-docker-compose.yaml](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/resources/init-resources/jupyter-docker-compose.yaml#L57)

Reply:
Yep, I'll try that